catalyst_voices/packages/internal/catalyst_voices_models/lib/src/registration/steps/create_base_profile_stage.dart

```diff
@@ -0,0 +1,29 @@
+/// Describes the base profile creation during registration.
+enum CreateBaseProfileStage {
+  /// Givens context to user about what is seed phrase.
+  instructions,
+
+  /// User inputs base profile data here. (email and display name).
+  setup,
+
+  /// Checkboxes with mandatory acknowledgements.
+  acknowledgements;
+
+  CreateBaseProfileStage? get next {
+    final isLast = index == CreateBaseProfileStage.values.length - 1;
```

Comment:
Imho this reads better
```suggestion
    final isLast = this == CreateBaseProfileStage.values.last;
```